src/runtime/crt/common/func_registry.c

```diff
@@ -101,7 +101,8 @@ tvm_crt_error_t TVMMutableFuncRegistry_Create(TVMMutableFuncRegistry* reg, uint8
 
   reg->registry.names = (const char*)buffer;
   buffer[0] = 0;  // number of functions present in buffer.
-  buffer[1] = 0;  // end of names list marker.
+  buffer[1] = 0;
```

Comment:
Better to explicitly mention the purpose. Something like

```suggestion
  buffer[1] = 0; // note that we combine the first two elements to form a 16-bit function index.
```